app/networkapi/settings.py

```diff
@@ -19,8 +19,8 @@
 # we rely on it being explicitly set (no default values) so that
 # we error out first.
 env = environ.Env(
-    DEBUG=(bool, False),
-    FILEBROWSER_DEBUG=(bool, False),
```

Comment:
If we do commit these defaults, we need to verify we turn them off on staging/production.